### PR TITLE
Updating if-demo functional-unit-time to match expected value

### PR DESCRIPTION
**Pull request by dcrreynolds**
### Types of changes

- [ ] Enhancement (project structure, spelling, grammar, formatting)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- Small update to one of the examples so the functional-unit-time value is accepted by the regex validation of the string. Otherwise an error `InputValidationError: "functional-unit-time" parameter is invalid. Error code: invalid_string.` is thrown.

### CODE REVIEW

**Comment by jmcook1186** [comment]
Hi - thanks for the PR - you can actually also delete the `functional-unit-duration` field altogether - it's no longer used.

Also, merging is blocked on DCO - if you agree with the DCO terms you can add a DCO remediation commit to unblock (see the instructions in the DCO details above)

**Comment by dcrreynolds** [comment]
Thank you for the feedback! I removed `functional-unit-duration`, added the missing version stamp, and made DCO happy.

**Review by jmcook1186 — APPROVED** [review]
great, thanks!

**Pull request merged**
